docs/fileformat-prose.md

```diff
@@ -283,20 +412,56 @@ We support IO with Pandas nullable integer and boolean arrays.
 We represent these on disk similar to `numpy` masked arrays, `julia` nullable arrays, or `arrow` validity bitmaps (see {issue}`504` for more discussion).
```

Comment:
Can't add the comment in the right place. I'd consider adding an example for "String array specification", maybe not in the doc and just in the actual downloadable Zarr file.